projects/authserver/ConnectionHandlers/ClientAuthConnectionHandler.h

```diff
@@ -24,7 +24,7 @@
 #pragma once
 
 #include <Networking/TcpServer.h>
-#include "..\Connections\AuthConnection.h"
+#include "../Connections/AuthConnection.h"
```

Comment:
Clang seems to have an issue with `\` includes, and this seems to be where most of the errors came from.

It may be a bit pedantic but I feel that changing all of the `\` to `/` should be in their own commit (or even pull request) as it is somewhat unrelated to the cross-platform work.

Reply:
This was has been addressed